realm/src/main/java/io/realm/exceptions/RealmError.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2015 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.exceptions;
+
+/**
+ * RealmError is Realm specific Error used when unrecoverable problems in the underlying storage engine. An RealmError
```

Comment:
"when unrecoverable problems in the underlying storage engine" -> "when unrecoverable problems **happen** in the underlying storage engine"